recipe/bld.bat

```diff
@@ -27,5 +27,4 @@ cmake --build . --target install --config Release
 if errorlevel 1 exit 1
 
 del %LIBRARY_BIN%\msvc*.dll
-del %LIBRARY_BIN%\Microsoft.*
 move %LIBRARY_LIB%\blosc.dll %LIBRARY_BIN%\
```

Comment:
@beckermr this seams to be the case now as well, and can be removed/causes your problem.

Not sure if we should keep the `concrt140.dll` and `msvcp140.dll` in `bin` or remove one (now) or both... probably both come with the MSVC compiler stack.

Reply:
I know nothing about windows so thanks a bunch for the tips! Feel free to commit the changes yourself too!

Reply:
You are welcome.
My Windows knowledge is also solely based on CI and conda-forge knowledge :)